personalizer.py

```diff
@@ -516,10 +516,9 @@ def _call(
             self.workspace.predict_one(parse_lines(text_parser, self.last_decision.vwtxt)))
         preds = {}
         for i, (j, actions) in enumerate(zip(self.last_decision.label.chosen, self.actions)):
-            preds[self.actions_map[i]] = str(actions[j]) 
-        llm_resp = super()._call(
-            run_manager=run_manager, inputs=preds, preds={}
-        )
+            preds[self.actions_map[i]] = str(actions[j])
+        inputs["preds"] = preds
```

Comment:
I believe it should be inputs = preds here (or just remove this line and pass preds as inputs in super._call)

Reply:
I am shoving everything in inputs so that we don't need to pass in extra args every time we want to do something new